### PR TITLE
[4.1] Add disable_native_system_stats_collector property

### PR DESCRIPTION
**Pull request by malcyon**
Changing `disable_sigar` flag to disable_native_system_stats_collector (see https://github.com/Graylog2/graylog2-server/pull/9840).

### CODE REVIEW

**Review by bernd — CHANGES_REQUESTED** [review]
We can only merge this once we have a 4.1 branch for the documentation. Setting this on hold until then.

**Comment by malcyon** [comment]
@bernd Are we ready for this PR to go in?

**Review by bernd — APPROVED** [review]
Yes! :+1:

**Pull request merged**
